flipperdevices/flipperzero-firmware
# [FL-1795] BLE GAP refactoring

# What's new

- Separate GAP and HCI
- Refactor GAP, battery, device info and serial services
- Support pairing and bonding
- Increase ble packet size
- Use os timer instead of RTC

# Verification 

- Build for f6 and f7 targets
- Verify pairing. Connect with BLE Client, verify pin code appearance. Verify disconnection after incorrect pin code passed
- Verify BLE speed by sending and receiving bytes from serial service. Expected speed ~ 4kB/s

# Checklist (do not modify)
